sdks/python/apache_beam/examples/cookbook/group_with_coder_test.py

```diff
@@ -80,22 +80,17 @@ def test_basics_without_type_check(self):
     # therefore any custom coders will not be used. The default coder (pickler)
     # will be used instead.
     temp_path = self.create_temp_file(self.SAMPLE_RECORDS)
-    group_with_coder.run([
```

Comment:
This means that the example will not run without type checking. The example doesn't mention type checking at all (in group_with_coder.py), so it's probably okay to assume people will be using the defaults.

Reply:
Correct. this was explicitly testing something (non-default) that is no longer supported.